{
  "Header": {
    "Original": "CVE-2025-39881 (In the Linux kernel, the following vulnerability has been resolved:  k ...)",
    "ID": "CVE-2025-39881",
    "Description": "(In the Linux kernel, the following vulnerability has been resolved:  k ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-4328-1}",
      "Type": "xref",
      "Bugs": [
        "DLA-4328-1"
      ]
    },
    {
      "Original": "- linux 6.16.8-1",
      "Type": "package",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.16.8-1"
    },
    {
      "Original": "[trixie] - linux 6.12.48-1",
      "Type": "package",
      "Release": "trixie",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.12.48-1"
    },
    {
      "Original": "[bookworm] - linux 6.1.153-1",
      "Type": "package",
      "Release": "bookworm",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.1.153-1"
    },
    {
      "Original": "[bullseye] - linux \u003cnot-affected\u003e (Vulnerable code not present)",
      "Type": "package",
      "Release": "bullseye",
      "Package": "linux",
      "Kind": "not-affected",
      "Description": "Vulnerable code not present"
    },
    {
      "Original": "NOTE: https://git.kernel.org/linus/3c9ba2777d6c86025e1ba4186dc5cd930e40ec5f (6.17-rc6)",
      "Type": "NOTE",
      "Description": "https://git.kernel.org/linus/3c9ba2777d6c86025e1ba4186dc5cd930e40ec5f (6.17-rc6)"
    }
  ]
}